platform/packages/finance/src/coin/dto/mod.rs

```diff
@@ -40,9 +43,22 @@ where
     // keep a Coin<C> in a Box<Member<G>> replacing all the struct member variables
     ticker: SymbolOwned,
     #[serde(skip)]
+    #[schemars(skip)]
     _g: PhantomData<G>,
 }
 
+impl<G> Clone for CoinDTO<G>
+where
+    G: Group,
```

Comment:
what is the reason the derived impl is not acceptable? The `Clone` requirement to G?

Reply:
Yes, indeed it is the bound on the generic. Those are, as mentioned above, logical constructs with no relation to the data itself. As already mentioned, this is unacceptable by the standards you laid out.

Reply:
let's refrain from implementing Close by ourselves since we're going to turn CoinDTO into Copy-able